.github/workflows/autobump.yml

```diff
@@ -406,18 +472,25 @@ env:
     mdbook
     mdcat
     mdzk
+    MeiliSearch
```

Comment:
Why the capitalisation?

Reply:
yeah, it does not need to be lower case, let me correct it.